modules/core/02-client/keeper/grpc_query.go

```diff
@@ -328,3 +328,55 @@ func (k Keeper) UpgradedConsensusState(c context.Context, req *types.QueryUpgrad
 		UpgradedConsensusState: protoAny,
 	}, nil
 }
+
+// VerifyMembership implements the Query/VerifyMembership gRPC method
+func (k Keeper) VerifyMembership(c context.Context, req *types.QueryVerifyMembershipRequest) (*types.QueryVerifyMembershipResponse, error) {
```

Comment:
Should we document this addition and the fact that there is going to be a set of default allowed queries made available to the Querier? Maybe in the 08-wasm docs?

Reply:
https://github.com/cosmos/ibc-go/issues/5846